ci/scripts/get-bitstream-strategy.sh

```diff
@@ -33,22 +33,31 @@ else
 fi
 
 # Retrieve the most recent bitstream at or before HEAD.
-BITSTREAM="--refresh HEAD" ./bazelisk.sh build ${bitstream_target}
+BITSTREAM="--refresh HEAD" ./bazelisk.sh build @bitstreams//:manifest
 
 # The directory containing the bitstream is named after the git hash.
-bitstream_file=$(ci/scripts/target-location.sh ${bitstream_target})
-bitstream_commit=$(basename "$(dirname ${bitstream_file})")
+manifest_file=$(ci/scripts/target-location.sh @bitstreams//:manifest)
 
-echo "Checking for changes against pre-built bitstream from ${bitstream_commit}"
-echo "Files changed:"
-git diff --stat --name-only ${bitstream_commit}
-echo
-echo "Changed files after exclusions applied:"
-# Use the cached bitstream if no changed files remain.
-if git diff --exit-code --stat --name-only ${bitstream_commit} -- "${excluded_files[@]}"; then
-  bitstream_strategy=cached
-else
+bitstream_commit=$(./util/py/scripts/get_bitstream_build_id.py \
```

Comment:
Shouldn't `util/py/scripts/get_bitstream_build_id.py` be invoked through Bazel? Or why are we adding it to `util/py/scripts/BUILD`?

Reply:
Yeah, it should. I bailed on that when I noticed bazel hiding access to workspace files that weren't in srcs, deps, or data. It was adding `@bitstreams//:manifest` that I felt unsure about, but on second thought, that ought to be fine.

Reply:
Updated to run from bazel. Thanks for poking me to "do it right" :)